templates/how-to-contribute.html

```diff
@@ -0,0 +1,365 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/bulma@0.9.4/css/bulma.min.css"
```

Comment:
Can you take example from other HTML files and use JINJA templates here instead of redeclaring the HTML block again.

Reply:
I've never worked with python and have first-time heard about JINJA Templates, so can you give me its docs to understand this?

I looked through other files and found out that they have included `{% extends 'base.html' %}` starting with every HTML file. I've tried to incorporate it into the "How to contribute" file but it's not working and showing [this](https://i.imgur.com/CvJs2tf.png) on the webpage.

P.S.: I'm not using a virtual environment or docker in the local environment, I'm using VSCode's live server extension, maybe it is a problem?

Reply:
Ok, so the error is because we need to put the content of body inside "content" block like this

```html
{% extends 'base.html' %}
{% block content %}

<!-- HTML for how to contribute page (just the content of `body` tag) -->

{% endblock content %}
```

As for running the code, I recommend using GitHub Codespaces, and follow the installation instructions as dictated in the README (use the virtual env approach).

I recommend going through a short tutorial on how Jinja templates work and their usage (https://codeburst.io/jinja-2-explained-in-5-minutes-88548486834e).

Reply:
Okay, I'm starting to understand JINJA Template. However, what about the Style and Script tags? Where should I put them? Above `{% extends 'base.html' %}`?

I've also given it a try by commenting out the `<head>`, `<style>`, `<script>` tag and replacing `<body>` with `{% block content %}` & `{% endblock content %}` but it's showing this [error](https://i.imgur.com/ld2AQjB.png). 

Now I have no idea what to do! Would it be okay if I added you as a contributor to my feature branch, so you can access & edit it in Codespaces?

Reply:
Hey @darshil-here were you able to fix the errors you were facing?

Reply:
Hey @Bhupesh-V! I couldn't give my attention last week because of my exams. Regarding the issues, the site worked fine on codespaces after your contribution 🙌

Reply:
Made the changes based on the reference provided by you 🙌